### PR TITLE
third hw (first) commit

### PR DESCRIPTION
**Pull request by Daria04**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw3/BaseTest.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+    protected WebDriver driver;
+
+    @BeforeClass()
+    public void oneTimeSetUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
```
Comment (DmitryKhodakovsky) [inline src/test/java/com/epam/tc/hw3/BaseTest.java]: Instance of the driver should be created before each test

File: src/test/java/com/epam/tc/hw3/test/SecondExerciseTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        headerComponent.serviceButtonClick();
+        ServiceComponent serviceComponent = PageFactory.initElements(driver, ServiceComponent.class);
+        serviceComponent.difElementsButtonClick();
+        DifferentElementsPage differentElements = PageFactory.initElements(driver, DifferentElementsPage.class);
+        differentElements.waterCheckboxClick();
+        differentElements.windCheckboxClick();
+        differentElements.radioItemClick();
+        differentElements.colorInputButtonClick();
+        differentElements.yellowButtonClick();
+        assertThat(differentElements.areLogRowsDisplayed());
```
Comment (DmitryKhodakovsky) [inline src/test/java/com/epam/tc/hw3/test/SecondExerciseTest.java:33]: Where is the checking of the text for the log row?

**Review by DmitryKhodakovsky — CHANGES_REQUESTED** [review]
Please fix issues

File: src/test/java/com/epam/tc/hw3/BaseTest.java (hunk 1 above)
Comment (Daria04) [reply src/test/java/com/epam/tc/hw3/BaseTest.java]: Instance of the driver is created before each test

File: src/test/java/com/epam/tc/hw3/test/SecondExerciseTest.java (hunk 2 above)
Comment (Daria04) [reply src/test/java/com/epam/tc/hw3/test/SecondExerciseTest.java:33]: I added necessary method and assert

**Review by DmitryKhodakovsky — APPROVED** [review]
HW3 approved

**Pull request merged**